Assets/Source/Entities/Behaviors/Commander.cs

```diff
@@ -7,9 +7,11 @@ public class Commander : MonoBehaviour
 {
 	// Get rid of these after testing movement
 	private GameObject tempCircle;
+	private Player player;
 
 	void Start()
 	{
+		player = transform.root.GetComponent<Player>();
```

Comment:
What would prevent us from keeping all the components for a unit in one game object?